terraform/scripts/build-langfuse-web-custom.sh

```
@@ -0,0 +1,86 @@
+TEMP_TAG="langfuse-web-custom-build"
+
+echo "Building Langfuse web image with custom base path:"
+echo "  Environment: $ENVIRONMENT"
+echo "  Base Path: $BASE_PATH" 
+echo "  Region: $AWS_REGION"
+echo "  ECR URI: $ECR_URI/$IMAGE_NAME:latest"
+
+# Clone and build
+echo "Cloning Langfuse repository..."
+git clone https://github.com/langfuse/langfuse.git
+cd langfuse
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)